spring-cloud-sleuth-stream/src/main/java/org/springframework/cloud/sleuth/stream/ServerPropertiesHostLocator.java

```diff
@@ -80,7 +83,7 @@ private String getAddress() {
 			address = this.serverProperties.getAddress().getHostAddress();
```

Comment:
ex here I guess we are getting this from properties, so curious why we need to make a public type for local address resolver (as opposed to package private utility). If it were a static utility method, we wouldn't need to affect and break the public ctor of ServerPropertiesHostLocator